library/src/main/java/com/bumptech/glide/Glide.java

```diff
@@ -383,6 +383,13 @@ public void trimMemory(int level) {
     }
 
     /**
+     * Clears disk cache.
+     */
+    public void clearDiskCache() {
```

Comment:
Add an assertion that this is on a background thread (https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/util/Util.java#L135), and a comment that says it must be called from a background thread.